components/src/preact/genomeViewer/loadGff3.ts

```diff
@@ -60,6 +60,13 @@ function getAttributes(attributes: string): Map<string, string> {
     return attrMap;
 }
 
+function removeQuotes(input: string | undefined): string | undefined {
+    if (!input) {
+        return undefined;
+    }
+    return input.replace(/^['"](.*)['"]$/, '$1');
+}
```

Comment:
I think this does the same.

```suggestion
function removeQuotes(input: string | undefined) {
    return input?.replace(/^['"](.*)['"]$/, '$1');
}
```

Reply:
I would also just use the input as a string and handle the undefined case before this function. So this would be 
```ts
function removeQuotes(input: string) {
    return input.replace(/^['"](.*)['"]$/, '$1');
}
```